.github/workflows/ci.yml

```diff
@@ -25,27 +25,24 @@ jobs:
         rust:
           - stable
           # MSRV
-          - 1.56.0
+          - 1.60.0
     steps:
       - uses: actions/checkout@v3
         with:
           persist-credentials: false
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@master
```

Comment:
Question: Shouldn't this be pinned to a SHA hash or version pin like others in this file?

Reply:
It's confusing, but the [documented](https://github.com/dtolnay/rust-toolchain#inputs) way to specify the toolchain version from a matrix:

> When passing an explicit toolchain as an input instead of `@rev`, you'll want to use "dtolnay/rust-toolchain@master" as the revision of the action.

Reply:
That is strange, but thanks for the documentation reference.